Vagrantfile

```diff
@@ -0,0 +1,27 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+    config.vm.provider "libvirt" do |libvirt, override|
+        libvirt.memory = 2048
+        libvirt.cpus = 3
+    end
+  config.vm.synced_folder ".", "/vagrant", type: "rsync",
+    rsync__exclude: ".tmp*"
+
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "fedora/24-cloud-base"
```

Comment:
Well, two quick reasons.  

1) If I understood you correctly, the CentOS and Fedora Atomic host required greater preparations than simply dnf install coverage and atomic.  And that is normally handled specially by your tools right which arent in play local use.

2. Aim small, miss small :)

In all seriousness, if you think I should pursue, I'm happy to do so.  I think the Makefile would then need to call a bash script instead of trying to handle this all in make.  Thoughts?